src/main/java/com/synopsys/integration/create/apigen/data/ClassCategories.java

```diff
@@ -177,6 +177,7 @@ private void populateData() {
         populate(RESPONSE, ClassSourceEnum.NULL, "AssignableUserView");
         populate(RESPONSE, ClassSourceEnum.NULL, "AssignedUserRequest");
         populate(RESPONSE, ClassSourceEnum.NULL, "ComponentVersionRemediatingView");
+        populate(RESPONSE, ClassSourceEnum.NULL, "ComponentVersionUpgradeGuidanceView");
```

Comment:
For some reason, the ComponentVersionUpgradeGuidanceView that extends BlackDuckComponent does not get deleted when we create the BlackDuckResponse.

Reply:
unless you delete the directory you previously generated, the other file will still be there.  since it is in a different package it will not be overrided.

Reply:
If you are supposed to delete the directory before running the generator, that should be documented somewhere.